lib/tracker.coffee

```diff
@@ -0,0 +1,48 @@
+# contants
+analyticsWriteKey = 'pDV1EgxAbco4gjPXpJzuOeDyYgtkrmmG'
+
+# imports
+_ = require 'underscore-plus'
+
+module.exports =
+class Tracker
+
+  constructor: (@analyticsUserIdConfig) ->
+    {allowUnsafeEval} = require 'loophole'
+    @pkg ?= require("../package.json")
+
+    # Setup Analytics
+    Analytics = null
+    allowUnsafeEval -> Analytics = require 'analytics-node'
```

Comment:
the costly operation is the allowUnsafeEval, so it makes sense to cache the import and call it only once